ios/titanium.xcconfig

```diff
@@ -4,8 +4,7 @@
 // OF YOUR TITANIUM SDK YOU'RE BUILDING FOR
 //
 //
-TITANIUM_SDK_VERSION = 9.3.2.GA
-
+TITANIUM_SDK_VERSION = 10.1.0
```

Comment:
Maybe use 10.0.2.GA, so most of the devs can immediately build from source without changing some meta files

Reply:
yes, 10.0.2.GA should be fine...